modules/admin_manual/pages/enterprise/authentication/kerberos.adoc

```diff
@@ -280,7 +290,8 @@ setspn -L <user-name>
 See the https://learn.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/cc731241(v=ws.11)[Microsoft setspn,window=_blank] documentation for details and more parameters. 
 --
 
-. Map the account `<user-name>` to the service principal `HTTP/<FQDN>@<REALM>` and generate a keytab file. To do so, open a command shell and type:
+. Map the account +
```

Comment:
Is this line an accidental left-over copy?

Reply:
No, it is just the main list item to see what the step is, as the others in the list. But the details are always in the next line if the line is too long. I fiddled a bit around to see what renders best and decided for this approach for ease of readbility. Just see the rendered version on staging.